application/account-management/WebApp/src/lib/apiErrorListSchema.ts

```diff
@@ -0,0 +1,39 @@
+import { getCamelCase } from "./getCamelCase.ts";
+import type { FetchResponse } from "openapi-fetch";
+import { z } from "zod";
+
+const ApiErrorListSchema = z.array(z.object({ code: z.string(), message: z.string() }));
+const ApiErrorSchema = z.object({
+  title: z.string(),
+  type: z.string(),
+  status: z.number(),
+  Errors: ApiErrorListSchema,
+});
+type ApiErrorList = z.infer<typeof ApiErrorListSchema>;
+
+export function getApiError(response: FetchResponse<any>) {
+  const { error = null } = response;
+  const validatedApiError = ApiErrorSchema.safeParse(error);
+  if (!validatedApiError.success) {
+    return {
+      title: "Unknown server error response",
+      status: 0,
+      type: "0",
+      Errors: [],
+    };
+  }
+  return validatedApiError.data;
+}
+
+export function getFieldErrors(apiErrorList: ApiErrorList) {
+  const fieldErrors: Record<string, string[]> = {};
+  apiErrorList.forEach((error) => {
+    const key = getCamelCase(error.code);
```

Comment:
Code from the api is Pascal cased (not matching the swagger docs)

Reply:
I guess this should be fixed on the server. But since it's just a string value it's not happening automatically.